queue_entry.py

```diff
@@ -26,7 +26,6 @@
 import time
 import queue_model_objects_v1 as queue_model_objects
 import os
-import ShapeHistory as shape_history
```

Comment:
Ops, I missed this one! Thanks to Ivars for the suggestion!


From: Ivars Karpics [mailto:notifications@github.com]
Sent: Monday, February 01, 2016 2:14 PM
To: mxcube/HardwareObjects
Cc: Jie Nan
Subject: Re: [HardwareObjects] a few fixes of mockups (#9)


In queue_entry.py<https://github.com/mxcube/HardwareObjects/pull/9#discussion_r51416085>:

> @@ -26,7 +26,6 @@

>  import time

>  import queue_model_objects_v1 as queue_model_objects

>  import os

> -import ShapeHistory as shape_history

this can be replaced with:
cpos = queue_model_objects.CentredPosition(pos_dict)
I also came to this issue when working with qt4 and the fix is in master branch (like 654)
there are other places where is shape_history and that needs a cleaning...

—
Reply to this email directly or view it on GitHub<https://github.com/mxcube/HardwareObjects/pull/9/files#r51416085>.